cci-stats/pkg/db/db.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE pipelines
```

Comment:
Is the file `db.sql` used anywhere here? Did you perhaps forget to include the migrator code?
If unused, should we keep it? It'll go stale.

Reply:
It's used to bootstrap the database. The database backing the stats on go/ci-dashboard was generated using this file. There is no migrator since the application is very simple.